packages/TorneloScoresheet/src/chessEngine/chessTsChessEngine.ts

```diff
@@ -99,30 +127,33 @@ const gameToPeiceArray = (game: Chess): ChessBoardPositions => {
   const board: BoardPosition[][] = [];
 
   // for each row
-  game.board().map((gameRow, rowIdx) => {
-    const row: BoardPosition[] = [];
-
-    // for each col in row
-    gameRow.map((peice, colIdx) => {
-      // no piece
-      if (peice === null) {
-        row.push({
-          piece: null,
-          position: boardIndexToPosition(rowIdx, colIdx),
-        });
-      }
-
-      // existing piece
-      else {
-        row.push({
-          piece: mapEnginePeice(peice),
-          position: boardIndexToPosition(rowIdx, colIdx),
-        });
-      }
-    });
+  game
+    .board()
+    .reverse()
+    .map((gameRow, colIdx) => {
+      const row: BoardPosition[] = [];
 
-    board.push(row);
-  });
+      // for each col in row
+      gameRow.map((peice, rowIdx) => {
+        // no piece
+        if (peice === null) {
+          row.push({
+            piece: null,
+            position: boardIndexToPosition(rowIdx, colIdx),
+          });
+        }
+
+        // existing piece
+        else {
+          row.push({
+            piece: mapEnginePeice(peice),
+            position: boardIndexToPosition(rowIdx, colIdx),
+          });
+        }
+      });
```

Comment:
Typo in piece.

Also, rather than an if else, I think you can do :

```ts
row.push({
  piece: piece === null ? null : mapEnginePiece(piece),
  position: ...
```

Usually prefer an expression over an if / else.